byterun/caml/mlvalues.h

```diff
@@ -108,19 +108,19 @@ bits  63        (64-P) (63-P)        10 9     8 7   0
 #define PROFINFO_MASK ((1ull << PROFINFO_WIDTH) - 1ull)
 
 #define Tag_hd(hd) ((tag_t) ((hd) & 0xFF))
-#ifndef WITH_SPACETIME
-#define Wosize_hd(hd) ((mlsize_t) ((hd) >> 10))
-#else
+#ifdef WITH_SPACETIME
 #define Hd_no_profinfo(hd) ((hd) & ~(PROFINFO_MASK << PROFINFO_SHIFT))
 #define Wosize_hd(hd) ((mlsize_t) ((Hd_no_profinfo(hd)) >> 10))
-#endif
+#else
+#define Wosize_hd(hd) ((mlsize_t) ((hd) >> 10))
+#endif /* SPACETIME */
 #ifdef ARCH_SIXTYFOUR
 /* [Profinfo_hd] is used when the compiler is not configured for Spacetime
    (e.g. when decoding profiles). */
 #define Profinfo_hd(hd) (((mlsize_t) ((hd) >> PROFINFO_SHIFT)) & PROFINFO_MASK)
 #else
 #define Profinfo_hd(hd) ((hd) & 0)
-#endif
+#endif /* ARCH_SIXTYFOUR */
 
```

Comment:
None of the above changes are needed for this GPR, is that right?  We should keep this minimal since it's for 4.04.